source/_integrations/sms.markdown

```diff
@@ -54,26 +54,66 @@ action:
       unicode: False
 ```
 
-### Manual confiration
+### Getting SMS messages
 
-To configure the notification service, edit your `configuration.yaml` file:
+You can also receive SMS messages that are sent to the SIM card number in your device.
+Every time there is a message received, `event: sms.incoming_sms` is fired with date, phone number and text message.
+Sample automation that forward all SMS to `user1`:
 
+#### Define a sensor in `configuration.yaml` to protect user phone number
 ```yaml
-notify:
-  - platform: sms
-    name: sms_person1
-    recipient: PHONE_NUMBER
-  - platform: sms
-    name: sms_person2
-    recipient: PHONE_NUMBER
+template:
+  - sensor:
+    - name: "User1 Phone Number"
+      state: !secret user1_phone_number
 ```
 
-### Getting SMS messages
+#### Define a script in `scripts.yaml` to use the sensor
 
-You can also receive SMS messages that are sent to the SIM card number in your device.
-Every time there is a message received, `event: sms.incoming_sms` is fired with date, phone number and text message.
+{% raw %}
+
+```yaml
+notify_sms_user1:
+  alias: Notify via SMS to User1
+  fields:
+    message:
+      description: The message content
+      example: The light is on!
```

Comment:
These are strings and should be enquoted to show the best practices (as per our YAML style guidelines)